src/main/groovy/org/transmartproject/core/biomarker/BioMarkerConstraint.groovy

```diff
@@ -0,0 +1,34 @@
+package org.transmartproject.core.biomarker
+
+/**
+ * Limits the results of {@link BioMarkerResource#retrieveBioMarkers(List<BioMarkerConstraint>)}
+ *
+ * Properties constraints are passed as map where key represents property name and value represents the value
+ * that property must be equal to. When map contains multiple properties constraints all of them must be met by
+ * bio marker entry to appear in the result. In other words it's assumed there is AND logical operation
+ * between property constraints.
+ * e.g. [type: 'PROTEIN', organism: 'HOMO SAPIENS'] limits result to the human proteins
```

Comment:
This paragraph is documentation for `PROPERTIES_CONSTRAINT`, so it should be in its doc block.